test/unit/ngsiv2/HTTP_receive_ngsiv2_measures-test.js

```diff
@@ -88,7 +88,13 @@ describe('HTTP: NGSIv2 Measure reception ', function () {
                         },
                         status: {
                             type: 'Text',
-                            value: 'connected'
+                            value: 'connected',
+                            metadata: {
```

Comment:
Maybe we should include also metadata in the NGSIv2 examples recently added to usermanual.md ?

Reply:
Done in https://github.com/telefonicaid/iotagent-json/pull/781/commits/9ae7d367cabaa85ac88530e8332d915f1dad585e